### ISSUE TITLE
Instruction scene

### ISSUE THREAD

**Issue by andrefarrugia96** [stevencamilleri/IMTAssignment2016#6]
When I entered the game I was checking the controls, and what I did not find was the instructions needed in order for the user to play the game. Meaning what needs to be collected and what needs to be avoided. Also it would be helpful that the instructions needed to pass to the next level would be added.

**Comment by stevencamilleri** [comment]
Makes sense to me, thanks Andre! I will make sure to add an instructions scene to show the user what to do and how to complete the game.
